i18n/locales/es-MX/hourofcode/promote/op-ed.md

```diff
@@ -10,7 +10,7 @@ nav: promote_nav
   
 
 
-Ir a [< % = resolve_url('code.org/promote') %>](%= resolve_url('https://code.org/promote') %) para encontrar información específica de estado sobre Ciencias de la computación.
+Go to [code.org/promote](%= codeorg_url('/promote') %) to find state specific information about computer science.
```

Comment:
Yeah, it looks like where we changed the function from `resolve_url` to `codeorg_url` it reset the translation. Any way we can keep these?

Reply:
Hmm, maybe. If `resolve_url` still works, I could undo the parts of https://github.com/code-dot-org/code-dot-org/pull/21466 that included those changes, and sync them up again but with the `update_without_changes` option.

Reply:
_does_ resolve_url still work? And if so, do you see any problems with that solution?

Reply:
Brendan made these updates when he removed the redirects to xx.code.org partner sites. `resolve_url` still works, but it's now for hourofcode.com urls only. Then we use `codeorg_url` to preserve the environment.

So we definitely want all the strings to have `codeorg_url` instead of `resolve_url` so it looks like we would need to do special work to keep the translation. I don't think this is high priority enough to sidetrack other i18n work, so we may have to settle for asking for a re-translation.